master/buildbot/www/websocket.py

```diff
@@ -169,6 +169,7 @@ def parse_hybi07_frames(buf):
         if header & 0x70:
             # At least one of the reserved flags is set. Pork chop sandwiches!
             raise WSException("Reserved flag in HyBi-07 frame (%d)" % header)
+            # XXX(sa2ajj): two lines below are DEAD DEAD DEAD
```

Comment:
You're correct that the code is dead -- I mentioned the origin of the code to indicate that, rather than improving it in place, we may be better off deleting the file and using the copy distributed with Twisted.  Sadly, [TW-4173](https://twistedmatrix.com/trac/ticket/4173), where the file originated, is still open.  So, perhaps best just left alone?